# HG changeset patch
# Parent 582d4a1c51f5ed9e5833b34ca70864ff512c4186
add PKG_MIRROR_MD5SUM to trigger download from mirror before svn checkout

diff --git a/utils/i2c-tools/Makefile b/utils/i2c-tools/Makefile
--- a/utils/i2c-tools/Makefile
+++ b/utils/i2c-tools/Makefile
@@ -17,6 +17,8 @@ PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VER
 PKG_SOURCE_VERSION:=r6204
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION)-$(PKG_SOURCE_VERSION).tar.gz
 
+PKG_MIRROR_MD5SUM:=xxx
+
 PKG_BUILD_PARALLEL:=1
 PKG_BUILD_DEPENDS:=PACKAGE_python-smbus:python
 
